tests/module/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`module_1.rb 1`] = `
+module ::A::B::C
+  module NestedModule::Foo
+    class ClassName
+      def add(x, y)
+        x + y
+      end
+    end
+  end
+end
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+module ::A::B::C
+  module NestedModule::Foo
+    class ClassName
+      def add(x, y)
+        x + y
+      end
+  end
+end
+end
```

Comment:
This indentation is slightly off, and I'm not quite sure why 🤔

Reply:
Hey, thanks for the PR.

This issue is due to dedent call in class and module.
If we replace it with just

```js
parts.push(indent(concat([hardline, body])), hardline, "end");
```

in both cases class and module, then it would resolve the issue.